OracleTuxedo/ART_SALT_TMA/samples/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use an official Tuxedo runtime as a base image
+FROM oracle/tuxedoartrttma:12.2.2
+# Add TMA cases to container, this tar ball will automatically be extracted by ADD command
+ADD tma_case.tar.gz /home/oracle/
+# Set user to ROOT
+USER root
+# Base image doesn't include ksh, make and x3270 add them, required by TMA sample cases
+RUN yum -y install x3270; yum -y clean all
```

Comment:
Combine the `RUN` directives into a single directive. Also, I'd suggest using `&&` between commands.